src/index.html

```diff
@@ -9,10 +9,45 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <h1 hidden>Product cards</h1>
```

Comment:
The 'h1' tag is hidden and does not provide any content to the user or for SEO purposes. Consider providing a visible heading that describes the content of the page for better accessibility and SEO.